regparser/builder.py

```diff
@@ -76,6 +119,7 @@ def gen_and_write_layers(self, reg_tree, act_info, cache, notices=None):
                 layer)
 
     def revision_generator(self, reg_tree):
+
```

Comment:
Assuming this is a question about the blank link, this is just a stray newline that I'm sure I put in by accident.